ext/date/php_date.stub.php

```diff
@@ -2,6 +2,165 @@
 
 /** @generate-class-entries */
 
+/**
+ * RFC4287, Section 3.3: http://www.ietf.org/rfc/rfc4287.txt
```

Comment:
I added the `@var` annotation for class constants for exactly this purpose. However, in case of global constants, declaring the type is generally not really useful (and gen_stub can infer the type for the manual based on the value).

On the other hand, annotating global constants with `@var` is useful for the docs in case the value is unknown, so I will add them when necessary.

Reply:
Yeah, it could be useful when the value is unknown and potentially helpful if the value is indirect (another constant).